routemaster/feeds.py

```diff
@@ -20,6 +21,11 @@ class FeedNotFetched(Exception):
     pass
 
 
+@functools.lru_cache()
```

Comment:
Using an explicit `maxsize` would be nice here, just saves a google to figure out what the connection pool size is.

Reply:
I've changed this to a thread local anyway, but unfortunately this probably wouldn't work as there's another layer of indirection within requests to the actual connection pool. There doesn't seem to be an easy way to expose the actual connection pool parameters here.